wiki/en/Installation-for-Linux.md

```diff
@@ -67,9 +71,19 @@ Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to conf
 
 Restart JACK for the new settings to take effect
 
+### Configure PipeWire
+
+Jamulus Clients need a JACK server included in PipeWire. It is automatically launched when Jamulus is started.
```

Comment:
My understanding is that PipeWire uses a compatibility layer to make JACK applications _think_ the JACK server is present (when it's not), so maybe this should be worded differently.

Reply:
PipeWire runs a "fake" JACK server with its own API and Library. From the client point, it is a JACK server.
The goal here is to say:
- You, the user, use the JACK parameter in Jamulus, whether you're using JACK or PipeWire.

Reply:
Ok, how about:

"The Jamulus Client requires a JACK sound server (native PipeWire support has not yet been implemented). However, PipeWire provides a compatibility layer that emulates the JACK server, so when you launch the Jamulus Client, PîpeWire automatically exposes itself to it as a JACK server."